packages/components/src/FormatFile/FormatFile.test.tsx

```diff
@@ -33,6 +33,26 @@ it("renders an image when provided as src", async () => {
   expect(await findByRole("img")).toBeInTheDocument();
 });
 
+it("renders a skeleton loader when provided an image", async () => {
+  jest.useFakeTimers();
```

Comment:
Do we need `useFakeTimers`, `runAllTimers` and `useRealTimers` in this test?

If I omit 3000 in the `setTimeout()` the test will still pass which feels strange to me?  Shouldn't that mean there is no loading time or am I reading this wrong...

Reply:
yeah, i'm actually rewriting that test. they are false positive. To answer your question, yes

Reply:
We good!